pkg/apis/batch/types.go

```diff
@@ -238,6 +239,20 @@ type CronJobList struct {
 	Items []CronJob
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CronJobManualRequest represents a manual triggering of a CronJob.
+type CronJobManualRequest struct {
```

Comment:
I'm still struggling with two options atm:

1. Take your current options which embeds the created job.
2. Just return `metav1.Status` from that endpoint.

Reply:
@kubernetes/api-reviewers any preference?